### PR TITLE
Slack Livestream Issues: Week 2 +

### LINKED ISSUE #91: Slack: The need for Livestream Issues
# Issue Name: Slack Bitcamp: Need for Livestream Issues
# Problem:
### Context:
The Slack Bitcamp has around 3-4 homework issues on top of a livestream for every week. These issues are meant to teach the campers the skills and concepts for the week. However, they are meant to build upon the important concepts introduced to campers via the livestream. If a student is not able to make a livestream for whatever reason, or requires additional help, it would be easier for them to refer to a written out tutorial as well as watching the livestream recording. This creates the need for an issue dedicated to outlining the contents of each week's livestream.

### Type:
Livestream Issue Documentation

# Actual Issue:
The need for Livestream Issues

# Desired Outcome:
Each week involves at least one guided project during livestream. The tutorial/guide for this project can be found under livestream in the respective weekly directories. The issue should lead the students to building the projects and understanding the concepts outlined in the livestream.

# (Optional) Hint/Example:
Refer to the [livestream issue for Week 1](https://github.com/bitprj/BitCamp/tree/master/Slack-Apps/week1/homework/responses) of BitCamp Slack as an example.

# Possible Solutions:
There are several resources outlining how to write a step by step coding tutorial. This [medium article](https://medium.com/handlebar-labs/how-to-create-tech-tutorials-7f3ecdc29f82) is a great place to start.

# Other Details:
None.
#90

### PR DESCRIPTION
**Pull request by FifiTeklemedhin**
Weeks 2+4

### CODE REVIEW

**Comment by lazyplatypus** [comment]
Hey there, we need to follow this format -> https://github.com/bitprj/BitCamp/tree/master/repo_logistics/sample-camp
for the README.md files.

**Comment by beaubhp** [comment]
@lazyplatypus what's wrong with README?

**Comment by lazyplatypus** [comment]
Look at the headers, summary doesn't exist for example. also make sure to add the homework folder asap

File: Slack-Apps/README.md [hunk 1]
```diff
@@ -20,10 +31,21 @@ In the Livestream Portion of Week 1, students will be introduced to the basics o
...
+- Creating a Slack App that uses `say()` and responds to a message.
+
+###  ❓ How will this help students?
+
+In Week 1, students will work towards the end goal of creating their first Slack App by being introduced to the skills necessary to create one (reading documentation, learning about NodeJS, etc). During the Livestream, students will be able to ask questions refering to their current task.
 
+### **Week 2**
 ***
 
+📚 **Summary**
```
Comment (lazyplatypus) [inline Slack-Apps/README.md]: this needs to be deleted.

**Review by lazyplatypus — APPROVED** [review]

**Pull request merged**